src/apple/system_profile.rs

```diff
@@ -54,6 +58,17 @@ impl DeveloperTools {
                 })
             },
         )
-        .map_err(Error::SystemProfilerFailed)?
+        .map_err(|ras_err| match ras_err {
```

Comment:
```suggestion
        .map_err(|err| match err {
```
There isn't any other place in cargo-mobile where we put a prefix before `err`; the type of the error is already indicated quite well by the match below.